FannyAndersson/PayWay
# Fix/registration

Hello!
Registration feature is complete to test.
How to test:

1. Try to register user. If success - you will be redirected to /login
2. Try to register one more user with the same phone number / email from case 1. Expected to receive error on submit
